trio/_core/_run.py

```diff
@@ -890,7 +898,9 @@ def __exit__(self):  # pragma: no cover
         assert False, """Never called, but should be defined"""
 
 
-def open_nursery(strict_exception_groups=None):
+def open_nursery(
+    strict_exception_groups: bool | None = None,
+) -> AbstractAsyncContextManager[Nursery]:
```

Comment:
why not ``typing.AsyncContextManager``?

Reply:
NB: `var: bool | None = None`  is redundant in that you can skip the `| None` part. Type checkers infer that from the `None` default.

Reply:
The `typing` version is simply an alias of the `contextlib` one after 3.9 / with `__future__.annotations` enabled.

Reply:
The `None` thing was originally the case, but more recently checkers have preferred that you do it explicitly, so there's less special cases in the rules.

Reply:
yeah it confused me a bit that `typing.ContextManager` is not `contextlib.contextmanager`, but rather `contextlib.AbstractContextManager`.
https://docs.python.org/3/library/typing.html#typing.ContextManager

Reply:
@Fuyukai Is the usage of this alias problematic for your tooling, or just a matter of curiosity?

Reply:
Nope, just curiosity.